src/components/notification/alert-message.ts

```diff
@@ -98,3 +116,47 @@ From: ${getMonikaInstance()}`
 
   return message
 }
+
+export const getMessageForStart = async (
+  hostname: string,
+  ip: string
+): Promise<NotificationMessage> => {
+  if (monikaInstance.length === 0) {
+    await getMonikaInstance(ip)
+  }
+
+  return {
+    subject: 'Monika is started',
+    body: `Monika is running on ${monikaInstance}`,
```

Comment:
Since `monikaInstance` now starts with the city, it should say `Monika is running from `

Reply:
done